packages/protocol/contracts/governance/ReleaseGoldInstance.sol

```diff
@@ -0,0 +1,558 @@
+   * @param walletAddress The wallet address to set for the account.
+   */
+  function setAccount(string calldata name, bytes calldata dataEncryptionKey, address walletAddress)
+    external
+    onlyWhenInProperState
+  {
+    getAccounts().setAccount(name, dataEncryptionKey, walletAddress);
+  }
+
+  /**
+   * @notice A wrapper setter function for creating an account.
+   */
+  function createAccount() external onlyCanVote onlyWhenInProperState {
+    require(getAccounts().createAccount(), "Account creation failed");
+  }
+
+  /**
+   * @notice A wrapper setter function for the name of an account.
+   * @param name A string to set as the name of the account.
+   */
+  function setAccountName(string calldata name) external onlyWhenInProperState {
+    getAccounts().setName(name);
+  }
+
+  /**
+   * @notice A wrapper setter function for the wallet address of an account.
+   * @param walletAddress The wallet address to set for the account.
+   */
+  function setAccountWalletAddress(address walletAddress) external onlyWhenInProperState {
+    getAccounts().setWalletAddress(walletAddress);
+  }
+
+  /**
+   * @notice A wrapper setter function for the for the data encryption key
+   *         and version of an account.
+   * @param dataEncryptionKey Secp256k1 public key for data encryption.
+   *                          Preferably compressed.
+   */
+  function setAccountDataEncryptionKey(bytes calldata dataEncryptionKey)
+    external
+    onlyWhenInProperState
+  {
+    getAccounts().setAccountDataEncryptionKey(dataEncryptionKey);
+  }
+
+  /**
+   * @notice A wrapper setter function for the metadata of an account.
+   * @param metadataURL The URL to access the metadata..
+   */
+  function setAccountMetadataURL(string calldata metadataURL) external onlyWhenInProperState {
+    getAccounts().setMetadataURL(metadataURL);
+  }
+
+  /**
+   * @notice Converts `account`'s pending votes for `group` to active votes.
+   * @param group The validator group to vote for.
+   * @dev Pending votes cannot be activated until an election has been held.
+   */
+  function activate(address group) external nonReentrant onlyWhenInProperState {
+    getElection().activate(group);
+  }
+
+  /**
+   * @notice Revokes `value` active votes for `group`.
+   * @param group The validator group to revoke votes from.
+   * @param value The number of votes to revoke.
+   * @param lesser The group receiving fewer votes than the group for which the vote was revoked,
+   *               or 0 if that group has the fewest votes of any validator group.
+   * @param greater The group receiving more votes than the group for which the vote was revoked,
+   *               or 0 if that group has the most votes of any validator group.
+   * @param index The index of the group in the account's voting list.
+   * @dev Fails if the account has not voted on a validator group.
+   */
+  function revokeActive(
```

Comment:
note that the revoke wrappers need to remain so that the release owner can revoke votes so that gold can be unlocked

Reply:
Note that participating in on-chain governance can also block unlocking gold [1]. I *think* things work as-is, since the release owner can authorize a different vote signer, and eventually the governance votes will expire and the gold can be unlocked, but please double check me here.


https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/governance/LockedGold.sol#L150

Reply:
Your logic seems sound, authorizing a new vote signer should overwrite any existing one, allowing you to get control of votes and unlock gold